rllib/utils/torch_utils.py

```diff
@@ -137,27 +142,22 @@ def clip_gradients(
 
     # Clip by global L2-norm (across all gradient tensors).
     else:
-        assert (
-            grad_clip_by == "global_norm"
-        ), f"`grad_clip_by` ({grad_clip_by}) must be one of [value|norm|global_norm]!"
         gradients_list = list(gradients_dict.values())
         total_norm = compute_global_norm(gradients_list)
         if len(gradients_list) == 0:
             return total_norm
         # We do want the coefficient to be in between 0.0 and 1.0, therefore
         # if the global_norm is smaller than the clip value, we use the clip value
         # as normalization constant.
-        device = gradients_list[0].device
-        clip_coef = grad_clip / torch.maximum(
-            torch.tensor(grad_clip).to(device), total_norm + 1e-6
```

Comment:
Creating the tensor here on CPU, then moving to GPU could be very expensive.
Replaced the entire logic with the simpler `torch.clamp()`, which works with a python float.